api/imapsync/update

```diff
@@ -43,28 +45,73 @@ if ($cmd eq 'update') {
     }
 
     umask 077;
-    open(FH, '>', "/var/lib/nethserver/secrets/imapsync-$input->{'name'}");
-    print FH $input->{'password'};
+    my ($login,$pass,$uid,$gid) = getpwnam('vmail');
```

Comment:
If we set permissions 01750 to `/var/lib/nethserver/imapsync/secrets/` and owner `root.vmail` the script does not need the `chown()` part

```suggestion
```

You can remove the previous `umask()` call.

Reply:
done